some conflict with uuid, complication error

```
   Compiling cdrs v2.2.1
     Running `rustc --edition=2018 --crate-name cdrs /root/.cargo/registry/src/github.com-1ecc6299db9ec823/cdrs-2.2.1/src/lib.rs --color always --crate-type lib --emit=dep-info,metadata,link -C opt-level=3 --cfg 'feature="default"' --cfg 'feature="v4"' -C metadata=541aeaedfd863e74 -C extra-filename=-541aeaedfd863e74 --out-dir /root/contrib/rust/server/target/release/deps -L dependency=/root/contrib/rust/server/target/release/deps --extern byteorder=/root/contrib/rust/server/target/release/deps/libbyteorder-86ca0f6d2ddd7023.rmeta --extern log=/root/contrib/rust/server/target/release/deps/liblog-5885e8f2c53c074d.rmeta --extern lz4_compress=/root/contrib/rust/server/target/release/deps/liblz4_compress-eb04c4a8c9eed48c.rmeta --extern r2d2=/root/contrib/rust/server/target/release/deps/libr2d2-7472572b246acab4.rmeta --extern rand=/root/contrib/rust/server/target/release/deps/librand-dc8994d085b5988a.rmeta --extern snap=/root/contrib/rust/server/target/release/deps/libsnap-f6027ff4766ef093.rmeta --extern time=/root/contrib/rust/server/target/release/deps/libtime-2dba67ce0bfa7751.rmeta --extern uuid=/root/contrib/rust/server/target/release/deps/libuuid-ef736748d326ada0.rmeta --cap-lints allow`
error[E0432]: unresolved import `uuid::BytesError`
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/cdrs-2.2.1/src/error.rs:10:5
   |
10 | use uuid::BytesError;
   |     ^^^^^^^^^^^^^^^^ no `BytesError` in the root

error[E0412]: cannot find type `BytesError` in crate `uuid`
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/cdrs-2.2.1/src/types/data_serialization_types.rs:179:66
    |
179 | pub fn decode_timeuuid(bytes: &[u8]) -> Result<uuid::Uuid, uuid::BytesError> {
    |                                                                  ^^^^^^^^^^ not found in `uuid`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0412, E0432.
For more information about an error, try `rustc --explain E0412`.
error: could not compile `cdrs`.
```

--- a/src/error.rs
+++ b/src/error.rs
@@ -7,7 +7,7 @@ use std::string::FromUtf8Error;
 
 use crate::compression::CompressionError;
 use crate::frame::frame_error::CDRSError;
-use uuid::BytesError;
+use uuid::Error as UUIDError;
 
 pub type Result<T> = result::Result<T, Error>;
 
@@ -21,7 +21,7 @@ pub enum Error {
     /// Internal IO error.
     Io(io::Error),
     /// Internal error that may be raised during `uuid::Uuid::from_bytes`
-    UUIDParse(BytesError),
+    UUIDParse(UUIDError),
     /// General error
     General(String),
     /// Internal error that may be raised during `String::from_utf8`
@@ -87,8 +87,8 @@ impl From<FromUtf8Error> for Error {
     }
 }
 
-impl From<BytesError> for Error {
-    fn from(err: BytesError) -> Error {
+impl From<UUIDError> for Error {
+    fn from(err: UUIDError) -> Error {
         Error::UUIDParse(err)
     }
 }

--- a/src/types/data_serialization_types.rs
+++ b/src/types/data_serialization_types.rs
@@ -175,8 +175,8 @@ pub fn decode_time(bytes: &[u8]) -> Result<i64, io::Error> {
     try_i_from_bytes(bytes)
 }
 
-// Decodes Cassandra `timeuuid` data (bytes) into Rust's `Result<uuid::Uuid, uuid::BytesError>`
-pub fn decode_timeuuid(bytes: &[u8]) -> Result<uuid::Uuid, uuid::BytesError> {
+// Decodes Cassandra `timeuuid` data (bytes) into Rust's `Result<uuid::Uuid, uuid::Error>`
+pub fn decode_timeuuid(bytes: &[u8]) -> Result<uuid::Uuid, uuid::Error> {
     uuid::Uuid::from_slice(bytes)
 }
 

